main.js

```diff
@@ -0,0 +1,53 @@
+//console.log("Hello")
+//console.log("enjoy")
+const firstSection = document.getElementById("section");
```

Comment:
The element you are looking for does not have an ID so `getElementById` will not work. What can you use instead?